Evaluation/Input/Content/Section2.1_Modeling_Strategy.md

```diff
@@ -1,6 +1,6 @@
-The general concept of building a PBPK model has previously been described by Kuepfer et al. ([Kuepfer 2016](#5-references)) Regarding the relevant anthropometric (height, weight) and physiological parameters (e.g. blood flows, organ volumes, binding protein concentrations, hematocrit, cardiac output) in adults was gathered from the literature and has been previously published ([PK-Sim Ontogeny Database Version 7.3](#5-references)). The information was incorporated into PK-Sim® and was used as default values for the simulations in adults.
+The general concept of building a PBPK model has previously been described by Kuepfer et al. ([Kuepfer 2016](#main-references)) Regarding the relevant anthropometric (height, weight) and physiological parameters (e.g. blood flows, organ volumes, binding protein concentrations, hematocrit, cardiac output) in adults was gathered from the literature and has been previously published ([PK-Sim Ontogeny Database Version 7.3](#main-references)). The information was incorporated into PK-Sim® and was used as default values for the simulations in adults.
```

Comment:
Again, using an anchor reference (here, the anchor `main-references` is defined in the **corresponding section of the qualification plan** (s. below)